1-js/06-advanced-functions/02-rest-parameters-spread/article.md

```diff
@@ -225,6 +225,49 @@ But there's a subtle difference between `Array.from(obj)` and `[...obj]`:
 So, for the task of turning something into an array, `Array.from` tends to be more universal.
 
 
+## Get a new copy of an object/array
+
+Remember when we talked about `Object.assign()` 
+[in the past](https://javascript.info/symbol#symbols-are-skipped-by-for-in) ?
+
+It is possible to do the exact same thing with the spread operator !
+
+```
+let arr = [1, 2, 3];
+let arrCopy = [...arr]; // spread the array into a list of parameters
+                        // then put the result into a new array
+
+// do the arrays have the exact same contents ?
+alert(JSON.stringify(arr) === JSON.stringify(arrCopy)); // true (same content)
+
+// are the arrays equal (they share the same reference) ?
+alert(arr === arrCopy); // false (not same reference)
+
+// modify our initial array:
+arr.push(4)
+alert(arr) // 1, 2, 3, 4
+alert(arrCopy) // 1, 2, 3
+```
+
+Note that it is possible to do the exact same to copy 
+of objects: 
+
+```
+let obj = { a: 1, b: 2, c: 3 };
+let objCopy = { ...obj }; // spread the object into a list of parameters
+                          // then return the result in a new object
+
+// do the objects have the exact same contents ?
+alert(JSON.stringify(obj) === JSON.stringify(objCopy)); // true (same content)
+
+// are the arrays equal (they share the same reference) ?
+alert(obj === objCopy); // false (not same reference)
+```
+
+This way of copying an object is much shorter than 
```

Comment:
Line 267 and 268 should be in one line?

Reply:
same as https://github.com/javascript-tutorial/en.javascript.info/pull/1823#discussion_r399063603